### ISSUE TITLE
Function `_http_view_func_wrapper` should use `@functools.wraps`

### ISSUE THREAD

**Issue by svleeuwen** [GoogleCloudPlatform/functions-framework-python#178]
For example when using Flask-Talisman you can specify per-view options using:

```
@functions_framework.http
@talisman(content_security_policy=None)
def foo(request):
    ...
```

Talisman will decorate the view function with a `talisman_view_options` attribute which is retrieved later like this:

```
view_options = getattr(view_function, 'talisman_view_options', {})
```

Adding `functools.wraps` makes sure all function attributes are copied from the original function to the `_http_view_func_wrapper`.

Will create a pull request for this in a minute.

**Comment by anniefu** [comment]
Hello, thanks for filing this issue!

This seems like a good change to add. Would it be possible for you to add a unit test and open a PR?

**Comment by svleeuwen** [comment]
@anniefu added test to [PR](https://github.com/GoogleCloudPlatform/functions-framework-python/pull/179)

**Comment by anniefu** [comment]
Fixed with #179
